### PR TITLE
add fasta support for load_genomic_breaks() function

### PR DESCRIPTION
**Pull request by johannesnicolaus**
This update adds functionality to load fasta instead of BSgenome objects, while maintaining the arguments for backwards compatibility

### CODE REVIEW

File: R/load_genomic_breaks.R [hunk 1]
```diff
@@ -78,17 +78,29 @@ load_genomic_breaks_GFF <- function (
...
   gb <- gb[gb$type == type]
   # Discard unused information
   gb$phase <- gb$Parent <- gb$Target <- gb$ID <- gb$source <- gb$type <- NULL
   # Convert query coordinates to GRanges
-  if (! is.null(query_bsgenome))
-    gb$query <- GRanges(gb$Name, seqinfo = seqinfo(query_bsgenome))
-  else
+  if (!is.null(query_bsgenome)) {
+    if (is.character(query_bsgenome)) {
+      query_seqinfo <- Seqinfo(readDNAStringSet(query_bsgenome))
```
Comment (charles-plessy) [inline R/load_genomic_breaks.R:96]: Should be `seqinfo` instead of `Seqinfo`

**Review by charles-plessy — CHANGES_REQUESTED** [review]
Thanks!  I think I found a typo, though.

**Comment by johannesnicolaus** [comment]
Fixed the typo. Should be OK now.

File: R/load_genomic_breaks.R (hunk 1 above)
Comment (johannesnicolaus) [reply R/load_genomic_breaks.R]: Typo is fixed in the next commit

**Comment by charles-plessy** [comment]
Just pushed it in version `0.14.6`.

**Pull request merged**